readme.md

```diff
@@ -38,13 +38,33 @@ This will require a significant rewrite of the papenmeier display driver.
 
 Used for BrlTTY.
 
-1. Download the latest `brltty-win-X.X-libusb.zip` from the [BrlTTY website](https://brltty.app/download.html) (not the 1.0 variant).

```

Comment:
I would suggest keeping both set of steps here

i.e. a subheading for building from scratch, and another from downloading from the website